core/services/relay/evm/event_binding.go

```diff
@@ -340,11 +337,38 @@ func (e *eventBinding) derefTopics(topics []any) error {
 	return nil
 }
 
+// makeTopics encodes and hashes params filtering values to match onchain indexed topics.
+func (e *eventBinding) makeTopics(params []any) ([]common.Hash, error) {
+	// make topic value for non-fixed bytes array manually because geth MakeTopics doesn't support it
```

Comment:
Is this a potential change we could propose for geth?

Reply:
I thought about it, I'll probably raise a PR when I find some time